### PR TITLE
Print complete oxo endpoint link.

### PR DESCRIPTION
**Pull request by ErebusZ**
image

### CODE REVIEW

**Review by benyissa — CHANGES_REQUESTED** [review]
I think we need to use console.info instead of print. since it has its own config.
image

File: setup.cfg [hunk 1]
```diff
@@ -113,6 +113,7 @@ serve =
     graphene-sqlalchemy
     cvss
     flask-cors
+    colorama
```
Comment (3asm) [inline setup.cfg:116]: colorama!!!

File: src/ostorlab/cli/serve/serve.py [hunk 2]
```diff
@@ -33,4 +34,7 @@ def serve(ctx: click.core.Context, host: str, port: int, refresh_api_key: bool)
             f"To authenticate, please use the following API key: {api_key.key}"
         )
     flask_app = app.create_app(graphiql=True)
+    print(
+        f"\n\t\t{colorama.Fore.GREEN}Serving on : {colorama.Style.RESET_ALL} http://{host}:{port}/oxo\n"
+    )
```
Comment (3asm) [inline src/ostorlab/cli/serve/serve.py:39]: Man, we use console for this!

**Review by 3asm — CHANGES_REQUESTED** [review]

File: src/ostorlab/cli/serve/serve.py [hunk 3]
```diff
@@ -33,4 +34,7 @@ def serve(ctx: click.core.Context, host: str, port: int, refresh_api_key: bool)
             f"To authenticate, please use the following API key: {api_key.key}"
         )
     flask_app = app.create_app(graphiql=True)
+    print(
```
Comment (3asm) [inline src/ostorlab/cli/serve/serve.py:37]: /graphql

**Review by 3asm — CHANGES_REQUESTED** [review]

**Comment by elyousfi5** [comment]
> I think we need to use console.info instead of print. since it has its own config. image
> 
> console.success(f"Serving on : http://{host}:{port}/oxo\n") image

+1

File: src/ostorlab/cli/serve/serve.py [hunk 4]
```diff
@@ -33,4 +33,5 @@ def serve(ctx: click.core.Context, host: str, port: int, refresh_api_key: bool)
             f"To authenticate, please use the following API key: {api_key.key}"
         )
     flask_app = app.create_app(graphiql=True)
+    console.success(f"Serving on : http://{host}:{port}/graphql")
```
Comment (oussamaessaji) [inline src/ostorlab/cli/serve/serve.py:36]: Can you add the second host

Comment (3asm) [inline src/ostorlab/cli/serve/serve.py:36]:
```suggestion
    console.success(f"Serving API on : http://{host}:{port}/graphql")
```

**Review by 3asm — DISMISSED** [review]

**Review by 3asm — APPROVED** [review]

**Pull request merged**